matrix.csv

```diff
@@ -46,11 +46,11 @@ fedora,29,2018.3,git,3
 fedora,29,2017.7,git,2
 
 # OPENSUSE
-opensuse/leap,15,develop,git,3
-opensuse/leap,15,2019.2,git,3
-opensuse/leap,15,2019.2,git,2
-opensuse/leap,15,2018.3,git,2
-opensuse/leap,15,2017.7,git,2
+opensuse/leap,15.1,develop,git,3
+opensuse/leap,15.1,2019.2,git,3
+opensuse/leap,15.1,2019.2,git,2
+opensuse/leap,15.1,2018.3,git,2
+opensuse/leap,15.1,2017.7,git,2
```

Comment:
Could even consider making the last one a `15.0`.

Reply:
Probably not making this one specifically `15.0` (for reverse compatibility), maybe adding an additional one.